climada/entity/impact_funcs/relative_cropyield.py

```diff
@@ -14,30 +14,33 @@
 """
 
 
-__all__ = ['IFCropPotential']
+__all__ = ['IFRelativeCropyield']
 
 import logging
 import numpy as np
 from climada.entity.impact_funcs.base import ImpactFunc
 
 LOGGER = logging.getLogger(__name__)
 
-class IFCropPotential(ImpactFunc):
+class IFRelativeCropyield(ImpactFunc):
     """Impact functions for agricultural droughts."""
 
     def __init__(self):
         ImpactFunc.__init__(self)
-        self.haz_type = 'CP'
+        self.haz_type = 'RC'
         self.intensity_unit = ''
         #self.continent = ''
 
     def set_relativeyield(self):
-        """Impact functions defining the impact as (intensity-1)"""
-        self.haz_type = 'CP'
+        """Impact functions defining the impact as intensity"""
+        self.haz_type = 'RC'
         self.id = 1
-        self.name = 'Crop Potential ISIMIP'
+        self.name = 'Relative Cropyield ISIMIP'
         self.intensity_unit = ''
-        self.intensity = np.arange(0, 11)
-        self.mdr = (self.intensity - 1)
-        self.mdd = (self.intensity - 1)
+        #intensity = 0 when the crop production is equivalent to the historical mean
+        #intensity = -1 for a complete crop failure
+        #intensity = 1 for a crop production surplus of 100%
+        self.intensity = np.arange(-1, 10)
```

Comment:
So the intensity range is from `-1 = failure`, to `10= 1000%` ? Any reason for the cap at `10`?

Reply:
Climada interpolates linearly after the last defined value, so we could also just define the impact function between -1 and 1. So the chosen value only defines e.g. the range plotted when visualising the impact function and therefore it might be good to keep a larger number than 1?

Reply:
This should be specified somewhere. The range is fine, and you could recall that a linear interpolation after is used in CLIMADA impact function.

Reply:
👍 
it's good to have the range cover the common stretch of intensity values as the extrapolation is a bit slower. at least in CLIMADA on Matlab that was the case